Src/AutoMoq/AutoMoqCustomization.cs

```diff
@@ -21,17 +35,33 @@ public AutoMoqCustomization()
         /// <see cref="MockRelay"/>.
         /// </summary>
         /// <param name="relay">The relay.</param>
+        [Obsolete("This constructor is obsolete and will be removed in the future version of the product. " +
```

Comment:
grammar: "the future version" hits the ear funny. I'd prefer either something like "the next major version" or "a future version". I wish I'd caught this when adding the same note to AutoFakeItEasy.

Reply:
Fixed. I didn't realized that `future version` is not exact enough to use the `the` article 😖